src/state/atoms/visibleBundlesAtom.test.ts

```
@@ -0,0 +1,97 @@
+describe('filterHiddenItems', () => {
+  it('should remove items with isHidden: true', () => {
+    const items: NavItem[] = [
+      { title: 'visible', href: '/a' },
+      { title: 'hidden', href: '/b', isHidden: true },
+      { title: 'also visible', href: '/c' },
+    ];
+    const result = filterHiddenItems(items);
+    expect(result).toEqual([
+      expect.objectContaining({ title: 'visible' }),
+      expect.objectContaining({ title: 'also visible' }),
+    ]);
```

ESLint

[error] Replace `⏎······expect.objectContaining({·title:·'visible'·}),⏎······expect.objectContaining({·title:·'also·visible'·}),⏎····` with `expect.objectContaining({·title:·'visible'·}),·expect.objectContaining({·title:·'also·visible'·})`

(prettier/prettier)